.nais/preprod.yml

```diff
@@ -57,7 +57,7 @@ spec:
         mountPath: /var/run/secrets/nais.io/vault
   env:
     - name: IDENT_ALLOW_LIST
-      value: "Z999999"
+      value: "Z999999,R154727,R156418,R158345,R158346,R160569,R165950,R165951,R165952,R150818,R150819,R155645,R162552,Z994123"
```

Comment:
Tenker vi kan fjerne `Z999999` fra listen, siden den bare ble lagt inn for å ha "noe" der

Reply:
Legg gjerne til Z990351 også :D Så kan jeg evt teste i preprod med min testident også :P

Reply:
La deg til i prod også 👍🏼